synapse/lib/storm.lark

```diff
@@ -65,8 +65,12 @@ CONTINUE: "continue"
 catchblock: "catch" _WSCOMM _valu _WSCOMM? "as" _WSCOMM? VARTOKN _WSCOMM? "{" query "}"
 trycatch: "try" _WSCOMM? "{" query "}" _WSCOMM? catchblock (_WSCOMM? catchblock)?
 
+_STOP: "stop"
+stop: _STOP
 _RETURN: "return"
 return: _RETURN _WS? "("[ _WS? _valu ] _WS?")"
+_EMIT: "emit"
+emit: _EMIT _WS? _valu
```

Comment:
is the `?` in this intentional?

Reply:
Def open to changing it ( should probably change it to _WSCOMM anyway ).  Do you think it should require a space? Most other things in our language are not space sensitive unless dictated by the grammar's ability to differentiate ( ie `foo:bar#baz` vs `foo:bar #baz` )